oracle/config/websocket.go

```diff
@@ -59,63 +59,63 @@ const (
 // WebSocketConfig defines a config for a websocket based data provider.
 type WebSocketConfig struct {
 	// Enabled is a flag that indicates whether the provider is websocket based.
-	Enabled bool `mapstructure:"enabled" toml:"enabled"`
+	Enabled bool `json:"enabled"`
```

Comment:
Wasn't the `Enabled` field mostly put in place to work around peculiarities w/ generating configs via toml? I'm thinking it doesn't make sense to have now--a provider will be enabled if there are feeds for it to fetch and disabled if not.